packages/ds-medicare-gov/src/styles/components/_SimpleFooter.scss

```diff
@@ -81,10 +74,12 @@
 }
 
 .m-c-footer__links {
-  margin-bottom: $spacer-3;
+  margin-block-end: $spacer-3;
 
-  @media screen and (min-width: $media-width-xl) {
-    margin-bottom: 0;
+  @media screen and (min-width: $media-width-lg) {
+    flex: 0 0 auto;
+    margin-block-end: 0;
+    margin-inline-end: $spacer-3;
```

Comment:
adding a buffer between the link blocks to ensure content doesn't bleed into each other. true, it's got `space-between` specified, but on some breakpoints that `space-between` is 0, so this is insurance the layout won't look jacked